src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/create-rgw-service-entities/create-rgw-service-entities.component.html

```diff
@@ -1,70 +1,125 @@
-          <span class="invalid-feedback"
-                *ngIf="createMultisiteEntitiesForm.showError('realmName', formDir, 'required')"
-                i18n>This field is required.</span>
-        </div>
+      <div class="form-item">
+        <cds-text-label
+          labelInputID="realmName"
+          [invalid]="   createMultisiteEntitiesForm.controls.realmName.invalid &&
+              (createMultisiteEntitiesForm.controls.realmName.touched ||
+                createMultisiteEntitiesForm.controls.realmName.dirty)"
+          [invalidText]="realmNameError"
+          i18n
+          >Realm Name
+          <input
+            cdsText
+            type="text"
+            id="realmName"
+            formControlName="realmName"
+            [invalid]="
+              createMultisiteEntitiesForm.controls.realmName.invalid &&
+              (createMultisiteEntitiesForm.controls.realmName.touched ||
+                createMultisiteEntitiesForm.controls.realmName.dirty)
+            "
+            [autofocus]="true"
+            modal-primary-focus
+          />
+        </cds-text-label>
+        <ng-template #realmNameError>
+          <span
+            class="invalid-feedback"
+            *ngIf="createMultisiteEntitiesForm.showError('realmName', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+        </ng-template>
       </div>
-      <div class="form-group row">
-        <label class="cd-col-form-label required"
-               for="zonegroupName"
-               i18n>Zone Group Name</label>
-        <div class="cd-col-form-input">
-          <input class="form-control"
-                 type="text"
-                 placeholder="Zone group name..."
-                 id="zonegroupName"
-                 name="zonegroupName"
-                 formControlName="zonegroupName">
-          <span class="invalid-feedback"
-                *ngIf="createMultisiteEntitiesForm.showError('zonegroupName', formDir, 'required')"
-                i18n>This field is required.</span>
-        </div>
+
+      <div class="form-item">
+        <cds-text-label
+          labelInputID="zonegroupName"
+          [invalid]="   createMultisiteEntitiesForm.controls.zonegroupName.invalid &&
+              (createMultisiteEntitiesForm.controls.zonegroupName.touched ||
+                createMultisiteEntitiesForm.controls.zonegroupName.dirty)"
+          [invalidText]="zoneGroupNameError"
+          i18n
+          >Zone Group Name
+          <input
+            cdsText
+            type="text"
+            placeholder="Zone group name..."
+            id="zonegroupName"
+            name="zonegroupName"
+            formControlName="zonegroupName"
+            [invalid]="
+              createMultisiteEntitiesForm.controls.zonegroupName.invalid &&
+              (createMultisiteEntitiesForm.controls.zonegroupName.touched ||
+                createMultisiteEntitiesForm.controls.zonegroupName.dirty)"
```

Comment:
you only need this. and same for all others
```suggestion
            [invalid]="
              createMultisiteEntitiesForm.controls.zonegroupName.invalid && createMultisiteEntitiesForm.controls.zonegroupName.dirty
```